demo_nodes_cpp/src/parameters/test_param_logic.cpp

```diff
@@ -0,0 +1,104 @@
+//
+// Created by deepanshu on 6/1/22.
+//
```

Comment:
The first part of this file should contain the license (you can copy the license section from other files in this repository).

You're welcome to list yourself as the author below the license, though this is not required.

Reply:
yeah, makes sense. I think my ide just entered that automatically. I will correct this:)